python/ruff/__main__.py

```diff
@@ -3,23 +3,30 @@
 import sysconfig
 from pathlib import Path
 
-RUFF_PATHS = [
-    Path(sysconfig.get_config_var("userbase")) / "bin" / "ruff",
-    Path(sysconfig.get_path("scripts")) / "ruff",
-]
-
 
 def find_ruff_bin() -> Path:
     """Return the ruff binary path."""
-    for ruff_path in RUFF_PATHS:
-        if ruff_path.is_file():
-            return ruff_path
-    raise FileNotFoundError(ruff_path)
+
+    path = Path(sysconfig.get_path("scripts")) / "ruff"
+    if path.is_file():
```

Comment:
Oh, but there’s a second problem: this doesn’t check the `.exe` extension on Windows.